tests/dinspec/secc/test_dinspec_secc_states.py

```diff
@@ -15,7 +15,7 @@ def get_extra_info(self, query_string: str):
         return "not supported"
 
 
-@patch("iso15118.shared.states.EXI.to_exi", new=Mock(return_value="\x01"))
+@patch("iso15118.shared.states.EXI.to_exi", new=Mock(return_value=b'01'))
 @pytest.mark.asyncio
 class TestEvseScenarios:
     @pytest.fixture(autouse=True)
```

Comment:
Looks like await was missed in two methods in this file:

```
    async def test_charging(self):
        current_demand: CurrentDemand = CurrentDemand(self.comm_session_mock)
        await current_demand.process_message(get_current_on_going_req())
        assert current_demand.next_state is None
        await current_demand.process_message(get_current_on_going_req())
        assert current_demand.next_state is None

    async def test_charging_finish(self):
        current_demand: CurrentDemand = CurrentDemand(self.comm_session_mock)
        await current_demand.process_message(get_current_on_going_req())
        assert current_demand.next_state is None
        await current_demand.process_message(get_current_on_going_req())
        assert current_demand.next_state is None
```

Reply:
yup...they passed because even when in sync they are None